strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/repository/group/BaseMavenGroupRepositoryComponentTest.java

```diff
@@ -81,4 +85,19 @@ public void removeRepositories()
         removeRepositories(getRepositories());
     }
 
+    protected void copyArtifactMetadata(String sourceRepositoryId,
+                                      String destinationRepositoryId,
+                                      String path)
```

Comment:
Cosmetic: Could you please re-indent this according to the convention? Thanks! :)